docs/green-lanes.md

```diff
@@ -0,0 +1,79 @@
+# Windsor Framework - Green Lanes
+
+The green lanes support exists to allow categorisation of Goods Nomenclatures according to the Windsor Framework agreement.
```

Comment:
Simplification:

The Green Lanes support allows the categorisation ...

Reply:
Problem is green lanes cannot explicitly categorise - it exists to provide data allowing the API client to categorise? I'll try and come up with a different phrasing

Reply:
Can we go with something like :

This API offers a capability for users to asses the categorization impact of Windsor Framework policies for past, current and future trades.

Reply:
Ended up with

| The green lanes support provides data allowing an API consumer to categorise Goods Nomenclatures according to the Windsor Framework agreement.